src/components/LocationPage/SocialButtons.tsx

```diff
@@ -52,7 +58,7 @@ const SocialButtons = props => {
         </TwitterShareButton>
       </SocialShareButton>
       <SocialShareButton {...buttonProps} color="#007fb1">
-        <LinkedinShareButton url={url} hashtags={[hashtag]} title={quote}>
+        <LinkedinShareButton url={url} title={quote}>
```

Comment:
hashtags is not valid LinkedinSharedButton interface, probably copied from the twitter one.